virtool/labels/db.py

```diff
@@ -0,0 +1,2 @@
+async def attach_sample_count(db, document, label_id):
```

Comment:
Needs docstring and typing.

Write specific tests for this function instead of relying on the integration tests for the API endpoints. In the API endpoint tests, you only test the case where each sample has one label. Add unit tests for this function that work with cases where a sample has zero or more labels.